src/content-page/views/ContentPage.tsx

```diff
@@ -18,8 +18,16 @@ interface ContentPageDetailProps extends DefaultSecureRouteProps {}
 const ContentPage: FunctionComponent<ContentPageDetailProps> = ({ match, user }) => {
 	const [t] = useTranslation();
 
+	const getCurrentPath = () => `/${(match.params as any).path}`;
+
 	// State
-	const [path] = useState<string>(`/${(match.params as any)['path']}`);
+	const [path, setPath] = useState<string>(getCurrentPath());
+
+	useEffect(() => {
+		if (path !== getCurrentPath()) {
+			setPath(getCurrentPath());
+		}
+	}, [match]);
```

Comment:
Warning: React Hook useEffect has missing dependencies: 'getCurrentPath' and 'path'. `match` mag weg.